ignite/pkg/cosmosgen/templates/module/module.ts.tpl

```diff
@@ -21,25 +22,19 @@ type {{ camelCase .Name }}Params = {
 };
 {{ end }}
 
-class Module extends Api<any> {
+class SDKModule extends Api<any> {
 	private _client: SigningStargateClient;
 	private _addr: string;
+	public registry;
 
-  	constructor(baseUrl: string) {
+	constructor(client: IgniteClient) {
 		super({
-			baseUrl
+			baseUrl: client.env.apiURL
 		})
+		this._client = client.client;
+		this._addr = client.env.rpcURL;
```

Comment:
Are these two properties actually being used? I could not find a reference to them in the `Api` and the `HttpClient` classes, neither in the generated code. Or are they defined here to be references later on?

Reply:
They are being referenced in the updated code.